src/views/tutorial-view/components/sidebar/index.tsx

```diff
@@ -0,0 +1,13 @@
+import { SidebarProps } from 'components/sidebar'
+import SidebarNav from 'components/sidebar/components/sidebar-nav'
+
+// @TODO match sidebar to spec
+export function TutorialSidebar({
+  title,
+  menuItems,
+}: Pick<
+  SidebarProps,
+  'title' | 'menuItems' | 'backToLinkProps'
+>): React.ReactElement {
+  return <SidebarNav title={title} menuItems={menuItems} />
```

Comment:
Still need to add the 'back to' and resources links but design is still figuring out _exactly_ how the sidebar should behave here so I'm leaving this for now.